guides-and-standards/standards--source-control.md

```diff
@@ -0,0 +1,153 @@
+- **Private by default**: Repositories should be private by default. If you need to make a repository public, please consult with the software team.
+- **Repository names:** Repository names are lower case and hyphenated. (e.g. `arcadia-software-handbook`). The names of analysis repositories should be prefixed with the year of the project (e.g. `2024-worm-tracking`).
+- **README**: Repositories should contain a descriptive README document.
+- **License**: Each repository should have a license file. The default license for Arcadia Science repositories is [the MIT license](https://github.com/Arcadia-Science/python-analysis-template/blob/main/LICENSE).
+- **Archiving repositories:** If a repository is no longer useful, make it read-only by archiving it. Follow the instructions [here](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories).
+- **No deleted repositories:** Never delete repositories. This ensures that other members (past and future) of Arcadia can see what worked and what hasn’t.
+
+### Public repositories
+
+Additional standards apply to public repositories to ensure that the code we release is reusable by others and that the computational artifacts we release are reproducible by others. These standards are described in a separate document [here](./standards--public-repos.md).
+
+### Repository templates
+
+We have Arcadia-specific templates for [Python](https://github.com/Arcadia-Science/python-analysis-template), [R](https://github.com/Arcadia-Science/r-analysis-template), [Snakemake](https://github.com/Arcadia-Science/snakemake-template), and [Nextflow](https://github.com/Arcadia-Science/nextflow-template-repository) repositories. It is strongly recommended to start new projects using these templates, as this will mean that your repository will automatically satisfy most of the requirements listed above.
+
+### Default repository settings
+
+Each repository should have the following settings. Please check this set of boxes when you create a new repo:
+
+<img width="768" alt="Default repository settings" src="https://user-images.githubusercontent.com/2692053/189210914-32052663-e2d3-4ab6-a418-1d1aca6a9afd.png">
+
+### [Optional] Code owners
+
+For projects with a lot of coordination between team members, it may make sense to define individuals or teams that are responsible for sections of code in a repository. This may make it easier to seek the appropriate code review before merging a pull request. You can accomplish this by using the [Code Owners](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) feature of Github by creating a `CODEOWNERS` file under the `.github` directory of your project. An example codeowners template file can be found [here](../.github/CODEOWNERS).
+
+## Branches and pull requests
+
+### Working on branches
+
+- We work on repositories using branches and not forks.
```

Comment:
we've had to fork external repos a couple times, idk if that's worth adding

Reply:
Added!